plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -65,6 +67,9 @@ protected void setup(Binder binder)
                 .to(GlueHiveMetastoreFactory.class)
                 .in(Scopes.SINGLETON);
 
+        Multibinder<RequestHandler2> requestHandlers = newSetBinder(binder, RequestHandler2.class, ForGlueHiveMetastore.class);
```

Comment:
Group along with https://github.com/trinodb/trino/blob/6849365d5a492a38de33404b3881959642c21cb3/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java#L52-L54 and do add this `RequestHandler2` binding as well to `requestHandlers`.